hermetic_build/library_generation/README.md

```diff
@@ -89,14 +89,6 @@ will be created/modified:
 | pom.xml (repo root dir)             | Always generated from inputs                                             |
 | versions.txt                        | New entries will be added if they don’t exist                            |
 
-### Change history
```

Comment:
IIUC this feature is not being actually removed. Should we keep it?

Reply:
Change history is not generated by `entry_point.py` so I think we don't need to include this part.

I plan to raise another PR to consolidate library generation and release note generation into a README in `hermetic_build` directory.

Reply:
Ah gotcha, I was missing the fact that this readme is in the `library_generation` folder.